CHANGE-NOTES.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased
+- Removed automatic rediscovery in case of empty machete file
```

Comment:
missing mention about repeated entry fix

Reply:
If you could piggyback that onto one of your PRs

Reply:
https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1282